### PR TITLE
Suppress the automated opening of the browser in developmental mode

### PR DESCRIPTION
**Pull request by IsmailM**
As requested in #146...

### CODE REVIEW

**Comment by IsmailM** [comment]
Had to create a separate method `compatible_linux_with_gui` to decrease the method's cyclomatic complexity to please Rubocop...

File: lib/sequenceserver.rb [hunk 1]
```diff
@@ -208,6 +208,10 @@ def using_ssh?
       true if ENV['SSH_CLIENT'] || ENV['SSH_TTY'] || ENV['SSH_CONNECTION']
     end
 
+    def compatible_linux_with_gui?
```
Comment (yeban) [inline lib/sequenceserver.rb]: Call it `has_xdg?`. There would be no point determining the presence of xdg-open command if the DISPLAY not set.

File: lib/sequenceserver.rb [hunk 2]
```diff
@@ -208,6 +208,10 @@ def using_ssh?
...
+      true if command?('xdg-open') || ENV['DISPLAY']
```
Comment (yeban) [inline lib/sequenceserver.rb]: Should it be `&&` instead of `||`?

File: lib/sequenceserver.rb [hunk 3]
```diff
@@ -196,9 +196,9 @@ def assert_blast_databases_present_in_database_dir
     end
 
     def open_default_browser(url)
-      return if using_ssh?
+      return if using_ssh? || verbose?
       if RUBY_PLATFORM =~ /linux/
```
Comment (yeban) [inline lib/sequenceserver.rb]: Can include the subsequent conditional here. `if RUBY_ ... && has_xdg?`.

**Comment by yeban** [comment]
> Had to create a separate method compatible_linux_with_gui to decrease the method's cyclomatic complexity to please Rubocop...

Good call.

**Comment by IsmailM** [comment]
Rubocop suggested `xdg?` over `has_xdg?`...

**Pull request merged**